cmake/modules/FindPythia8.cmake

```diff
@@ -22,3 +22,7 @@ include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Pythia8
   REQUIRED_VARS PYTHIA8_LIB_DIR PYTHIA8_INCLUDE_DIR
 )
+
+if(Pythia8_FOUND)
+  list(APPEND LD_LIBRARY_PATH ${PYTHIA8_LIB_DIR})
```

Comment:
What about Mac?

Reply:
It is just an internal variable. But this will change again. I don't like having to collect the lib dirs like this.